gizmo.py

```diff
@@ -0,0 +1,31 @@
+# hello function
+def hello(name, country = "Finland"):
+    print("Hello", name + ", how are things in", country + "?")
+# spelling function
+def spell(word):
+    for letter in word:
+        print (letter + '.', end = '')
+        
+# editing strings
+def relative_path(arr):
+    for i in arr:
+        i = './subjects/mock_recording_'+i+'.rec'
```

Comment:
You first use `i` as iterator variable, but then overwrite it to be a string. This works, but will confuse anyone (including your future self) trying to read this code. I recommend using a different name for the string variable.